  0%|          | 0/18 [00:00<?, ?it/s]  6%|▌         | 1/18 [02:33<43:22, 153.11s/it] 11%|█         | 2/18 [05:56<48:39, 182.44s/it] 17%|█▋        | 3/18 [08:05<39:35, 158.38s/it] 22%|██▏       | 4/18 [08:56<27:03, 115.99s/it] 28%|██▊       | 5/18 [09:56<20:44, 95.73s/it]  33%|███▎      | 6/18 [11:42<19:50, 99.19s/it] 39%|███▉      | 7/18 [15:04<24:19, 132.64s/it] 44%|████▍     | 8/18 [16:52<20:48, 124.89s/it] 50%|█████     | 9/18 [19:25<20:03, 133.72s/it] 56%|█████▌    | 10/18 [20:37<15:16, 114.58s/it] 61%|██████    | 11/18 [22:20<12:57, 111.06s/it] 67%|██████▋   | 12/18 [22:35<08:11, 81.90s/it]  72%|███████▏  | 13/18 [35:20<24:03, 288.78s/it] 78%|███████▊  | 14/18 [36:45<15:09, 227.35s/it] 83%|████████▎ | 15/18 [40:12<11:03, 221.05s/it] 89%|████████▉ | 16/18 [40:36<05:23, 161.81s/it] 94%|█████████▍| 17/18 [41:02<02:01, 121.03s/it]100%|██████████| 18/18 [41:36<00:00, 94.74s/it] 100%|██████████| 18/18 [41:36<00:00, 138.67s/it]
Traceback (most recent call last):
  File "evaluateBias_dexperts.py", line 188, in <module>
    main(args)
  File "evaluateBias_dexperts.py", line 101, in main
    with open(os.path.join(args.out_dir, args.model + '_' + filename), "w") as outfile:
AttributeError: 'Namespace' object has no attribute 'model'
